src/nunit.analyzers/UpdateStringFormatToInterpolatableString/UpdateStringFormatToInterpolatableStringAnalyzer.cs

```diff
@@ -37,7 +37,8 @@ public sealed class UpdateStringFormatToInterpolatableStringAnalyzer : BaseAsser
 
         protected override bool IsAssert(bool hasClassicAssert, IInvocationOperation invocationOperation)
```

Comment:
```suggestion
        protected override bool IsAssertOrAssume(bool hasClassicAssert, IInvocationOperation invocationOperation)
```

Reply:
You cannot change the name of a method in an `override`.
I'm not going to change the base class name and method for this.